### PR TITLE
updating home page with finished graphics

### PR DESCRIPTION
**Pull request by mansona**
This is just splitting @MelSumner 's work from #558 into different PRs 👍 

Fixes https://github.com/ember-learn/ember-website/issues/384

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -54,7 +56,8 @@
     </div>
     <div class="layout mt-4">
       <div class="lg:col-4 lg:start-3">
-        <img class="img-fluid" src="https://dummyimage.com/600x400/f47bf6/fff" alt="">
+        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424"
+        srcset="/images/home/ember-data-2x.png 2x">
```
Comment (MelSumner) [inline app/templates/index.hbs]:
```suggestion
        >
```

File: app/templates/index.hbs [hunk 2]
```diff
@@ -54,7 +56,8 @@
     </div>
     <div class="layout mt-4">
       <div class="lg:col-4 lg:start-3">
-        <img class="img-fluid" src="https://dummyimage.com/600x400/f47bf6/fff" alt="">
+        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424"
```
Comment (MelSumner) [inline app/templates/index.hbs]:
```suggestion
        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424">
```

Comment (MelSumner) [reply app/templates/index.hbs]: the 2x png for this graphic is off

Comment (mansona) [reply app/templates/index.hbs]: ah yes I remember you said! I'll fix it now 🎉

**Pull request merged**